modules/t/test-genome-DBs/homo_sapiens/gene2phenotype/meta.txt

```diff
@@ -17,5 +17,6 @@
 20	\N	patch	patch_104_105_k.sql|adding decreased gene product level as a mutation consequence to support cardiac
 15	\N	patch	patch_105_106_a.sql|schema version
 21	\N	patch	patch_106_107_a.sql|schema version
+22	\N	patch	patch_106_107_b.sql|drop RD and IF con
```

Comment:
You can write `confidence` instead of `con`:
`drop RD and IF confidence values`